src/about/index.ts

```diff
@@ -0,0 +1,406 @@
+
+import {
+  h, VNode
+} from 'phosphor/lib/ui/vdom';
+
+import {
+  VDomModel, VDomWidget
+} from '../common/vdom';
+
+/**
+ * The id name added to the About plugin root DOM node.
+ */
+const ABOUT_ID = 'about';
+
+/**
+ * The class name added to each page in the About plugin.
+ */
+const SECTION_CLASS = 'jp-About-section';
+
+/**
+ * The class name added to center content.
+ */
+const SECTION_CENTER_CLASS = 'jp-About-sectioncenter';
+
+/**
+ * The class name added to group elements on the initial page.
+ */
+const CONTAINER_CLASS = 'jp-About-container';
+
+/**
+ * The class name added to elements in a row.
+ */
+const ROW_CLASS = 'jp-About-row';
+
+/**
+ * The class name added to elements in a column.
+ */
+const COLUMN_CLASS = 'jp-About-column';
+
+/**
+ * The class name used specify postion of elements.
+ */
+const HALF_CLASS = 'jp-About-one-half';
+
+/**
+ * The class name added to the About plugin description on the intial page.
+ */
+const DESC_ONE_CLASS = 'jp-About-desc-one';
+
+/**
+ * The class name added to other plugin descriptions on the intial page.
+ */
+const DESC_TWO_CLASS = 'jp-About-desc-two';
+
+/**
+ * The class name added to headers of other plugins on the intial page.
+ */
+const DESC_TWO_HEADER_CLASS = 'jp-About-desc-two-header';
+
+/**
+ * The class name added to headers.
+ */
+const HEADER_CLASS = 'jp-About-header';
+
+/**
+ * The class name added to plugin pages.
+ */
+const CONTENT_CLASS = 'jp-About-content';
+
+/**
+ * The class name added to descriptions on plugin pages.
+ */
+const CONTENT_DESC_CLASS = 'jp-About-content-desc';
+
+/**
+ * The class name added to the navigation down button.
+ */
+const NAV_CLASS = 'jp-About-nav-button';
+
+/**
+ * The class name added to all images and icons.
+ */
+const IMAGE_CLASS = 'jp-img';
+
+/**
+ * The class name added to the JupyterLab logo.
+ */
+const LOGO_CLASS = 'jp-About-logo';
+
+/**
+ * The class name added to the main area icon.
+ */
+const MAIN_AREA_ICON_CLASS = 'jp-About-hero-mainarea';
+
+/**
+ * The class name added the command palette icon.
+ */
+const COMMAND_ICON_CLASS = 'jp-About-hero-command';
+
+/**
+ * The class name added to the filebrowser icon.
+ */
+const FILEBROWSER_ICON_CLASS = 'jp-About-hero-filebrowser';
+
+/**
+ * The class name added for the notebook icon.
+ */
+const NOTEBOOK_ICON_CLASS = 'jp-About-hero-notebook';
+
+/**
+ * The class name added to the main area image.
+ */
+const MAIN_AREA_IMAGE_CLASS = 'jp-About-mainarea';
+
+/**
+ * The class name added to the command palette image.
+ */
+const COMMAND_IMAGE_CLASS = 'jp-About-command';
+
+/**
+ * The class name added to the filebrowser image.
+ */
+const FILEBROWSER_IMAGE_CLASS = 'jp-About-fb';
+
+/**
+ * The class name added to the notebook image.
+ */
+const NOTEBOOK_IMAGE_CLASS = 'jp-About-nb';
+
+export
+class AboutModel extends VDomModel {
+  constructor() {
+    super();
+    this._title = 'Welcome to the JupyterLab Alpha preview';
+    this._headerText = [
```

Comment:
We can avoid having the private variables by setting the public values here and having `readonly title: string;` etc. on the class.